src/Plugin/resource/Field/ResourceFieldEntity.php

```diff
@@ -1020,6 +1021,16 @@ public function isComputed() {
   /**
    * {@inheritdoc}
    */
+  public function autoDiscovery() {
+    if (method_exists($this->decorated, 'autoDiscovery')) {
```

Comment:
There is none, but I don't want to enforce it for new resource fields that implementors (and related modules) can create.